test/fhirdefs/FHIRDefinitions.test.ts

```diff
@@ -0,0 +1,592 @@
+import { loadFromPath } from '../../src/fhirdefs/load';
+import { FHIRDefinitions } from '../../src/fhirdefs/FHIRDefinitions';
+import path from 'path';
+import { Type } from '../../src/utils/Fishable';
+
+describe('FHIRDefinitions', () => {
+  let defs: FHIRDefinitions;
+  beforeAll(() => {
+    defs = new FHIRDefinitions();
+    loadFromPath(
+      path.join(__dirname, '..', 'testhelpers', 'testdefs', 'package'),
+      'test#1.1.1',
+      defs
+    );
+  });
+
+  describe('#fishForFHIR()', () => {
+    it('should find base FHIR resources', () => {
+      const conditionByID = defs.fishForFHIR('Condition', Type.Resource);
```

Comment:
So this file will fail if you delete the definitions from the `testdefs/package` folder, but I guess there is probably really no way around that, since we are trying to test the fishing we don't want to use `TestFisher`. It's probably not a big deal overall, but it is something to be aware of.

Reply:
Oh right.  We could do like we did in the load test file and create a `TestFisher` in the setup and just manually fish for all the things we know we'll need.  I missed that here.